docs/docs/06_plugins/01_available-plugins/01-transactional/index.md

```diff
@@ -269,29 +270,29 @@ class AccountService {
 
 The `ClsPluginTransactional` constructor takes an options object with the following properties:
 
--   **_`imports`_**`: any[]`  
+-   **_`imports`_**`: any[]`
```

Comment:
Please do not remove the double spaces from ends of these lines (or make your linter not do it automatically). They trigger line wrap.